apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/CoverageTransformerV2.java

```diff
@@ -342,6 +345,24 @@ private Coverage transformPartC(Beneficiary beneficiary, Profile profile) {
       transformEntitlementDualEligibility(coverage, beneficiary);
     }
 
+    String contractAndPbpID =
+        concatenateContractAndPbp(
+            getPartCContractNumber(beneficiary), getPartCPbpNumber(beneficiary));
+
+    if (profile.equals(Profile.C4DIC)) {
+      // Hide resource Part C when Contract ID or PBP ID is null
+      if (contractAndPbpID == null) {
+        return null;
+      }
+      createCoverageClass(coverage, CoverageClass.PLAN, contractAndPbpID, Optional.empty());
+    } else {
+      createCoverageClass(
```

Comment:
Looks like the group is still missing here.

Reply:
good catch